recipes/xmatchview/meta.yaml

```diff
@@ -0,0 +1,30 @@
+
+package:
+    name: xmatchview
+    version: v1.1.1
+
+source:
+  url: https://github.com/bcgsc/xmatchview/archive/v1.1.1.tar.gz
+  md5: a1fb9eb41ba0d8c0a251f99433de1b7c
+
+build:
+    noarch: python
+    number: 0
+
+requirements:
+    build:
+        - python 2.7
+        - PIL
+
+    run:
+        - python 2.7
+        - PIL
+
+test:
+    imports:
+       - PIL
+
+about:
+    summary: Genome sequence alignment visualization
+    home: http://www.bcgsc.ca/platform/bioinfo/software/xmatchview
+    license: GNU General Public License v3.0
```

Comment:
GPL requires that the license be distributed with the software. There should be a `LICENSE.txt` or so in the package. Just put `license_file: NAME_OF_FILE` here.

Also consider adding `doc_url` and `dev_url` if they are available.

If you would like to become the responsible maintainer for this recipe, also add

```
extra:
  recipe-maintainers:
    - zyxue
```

Reply:
LICENSE file is inside the package, https://raw.githubusercontent.com/bcgsc/xmatchview/master/LICENSE

Reply:
Ok, then just put `license_file: LICENSE`. The `setup.py` won't copy it into the installation folder. By adding the line, `conda-build` knows to add the file to the package to comply with the license terms.